simSPI/linear_simulator/params_utils.py

```diff
@@ -0,0 +1,352 @@
+"""contains functions and classed for parameter generation."""
+from abc import ABCMeta, abstractstaticmethod
+
+import numpy as np
+import starfile
+import torch
+import torch.fft
+from pytorch3d.transforms import (
+    euler_angles_to_matrix,
+    matrix_to_euler_angles,
+    random_rotations,
+)
+
```

Comment:
looks like really good stuff!
https://youtu.be/Pph1r-x9nyY

Reply:
are you sure you can install it through pip OK? https://youtu.be/Pph1r-x9nyY?t=96

Reply:
Yes, I installed it using pip. It has been put in the pip dependency in the new environement.yml and I was able to create and activate that environment.